src/sage/schemes/elliptic_curves/kraus.py

```diff
@@ -746,12 +753,12 @@ def check_Kraus_global(c4, c6, assume_nonsingular=False, debug=False):
 
     - ``c4``, ``c6`` -- elements of a number field
 
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
-      check for integrality and nosingularity.
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Either False if Kraus's conditions fail, or, if they pass, an
+    Either ``False`` if Kraus's conditions fail, or, if they pass, an
     elliptic curve E which is integral and has c-invariants c4,c6.
```

Comment:
```suggestion
    elliptic curve `E` which is integral and has c-invariants `c_4`,`c_6`
```

Reply:
That would have to be either
```
 `c_4,c_6`
or
```
 `c_4`, `c_6`
```

Follow-up PR?